### PR TITLE
feat(sdk): chained entity path on nested tasks

### PR DESCRIPTION
**Pull request by OzBenSimhonTraceloop**
- [x] I have added tests that cover my changes.
- [ ] If adding a new instrumentation or changing an existing one, I've added screenshots from some observability platform showing the change.
- [ ] PR name follows conventional commits format: `feat(instrumentation): ...` or `fix(instrumentation): ...`.
- [ ] (If applicable) I have updated the documentation accordingly.

### Example
Assuming we have a workflow with three nesting levels of tasks
```
some_workflow 
=> outer_task 
    => inner_task
         =>  inner_inner_task
```

The expected entity paths will be 
- `some_workflow` path is None
- `outer_task` path is 'outer_task'
- `inner_task` path is 'outer_task.inner_task'
- `inner_inner_task` path is 'outer_task.inner_task.inner_inner_task'

### CODE REVIEW

File: packages/sample-app/sample_app/methods_decorated_app.py [hunk 1]
```diff
@@ -10,65 +10,32 @@
 Traceloop.init(app_name="joke_generation_service")
 
 
-@task(name="joke_creation", version=1)
```
Comment (nirga) [inline packages/sample-app/sample_app/methods_decorated_app.py:13]: Why did you delete this?

**Review by nirga — COMMENTED** [review]
That's a breaking change, let's wait with it for later this week as I want to release a small fix today.
Left a nit comment

File: packages/sample-app/sample_app/methods_decorated_app.py (hunk 1 above)
Comment (OzBenSimhonTraceloop) [reply packages/sample-app/sample_app/methods_decorated_app.py:13]: Actually I copied this file from Gal branch, I just saw he excluded it from his merged PR, so I'll remove this change as well 🙈

**Review by nirga — APPROVED** [review]

**Review by galkleinman — APPROVED** [review]
